docs/setup_instructions/windows_installation_instructions.md

```diff
@@ -246,14 +259,26 @@ You may see some warnings as the installation proceeds related to `timedatectl`,
 After installation is complete, you can exit the R console by typing `q()` and pressing Enter.
 You will be asked whether to save your workspace, and we recommend choosing `n`.
 
-## GitKraken
 
-We will use the GUI (graphical user interface) called `GitKraken` to work with Git in this workshop.
+### Installing Pandoc on WSL 2
 
+As part of this workshop, we'll be working with [R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).
+To render these documents to HTML from the command line, you will need a separate piece of software called Pandoc.
+(Note that you do not need a separate installation of Pandoc to render these documents from _within RStudio_ - only to render from the command line.)
 
+To install Pandoc on the _WSL 2_ side of your machine, enter this command in the Ubuntu terminal:
 
-GitKraken can be installed from <https://www.gitkraken.com>, where you can click "Download GitKraken Desktop Free" (_do not pay!!_):
+```sh
+sudo apt-get install pandoc
+```
+
+You will then be prompted for your password, and Pandoc will install.
```

Comment:
Do we want to just roll this into the rest of the installs at the top? I don't think that we should need a separate step here. 
```suggestion
You may then be prompted for your password, and Pandoc will install.
```